### ISSUE TITLE
This TODO has been done?

### ISSUE THREAD

**Issue by holyseven** [yassouali/pytorch_segmentation#88]
I believe this TODO has been done?

https://github.com/yassouali/pytorch_segmentation/blob/master/models/pspnet.py#L44
TODO: Use synch batchnorm

**Comment by yassouali** [comment]
Hi @holyseven 

Yes, you are correct, thanks for the remark, just removed it.
